ggml/src/ggml-hip/CMakeLists.txt

```diff
@@ -40,6 +40,9 @@ find_package(hip     REQUIRED)
 find_package(hipblas REQUIRED)
 find_package(rocblas REQUIRED)
 
+# Workaround old compilers
```

Comment:
please move this down a bit as the find_package calls and the version check below are logically related operations